src/app/game/snakeladder/page.tsx

```
@@ -0,0 +1,96 @@
+    setGameStatus("Player 1's turn");
+    setMoves([]);
+  };
+
+  return (
+    <div className={styles.container}>
+      <h1 className={styles.title}>Snake & Ladder</h1>
+      <div className={styles.gameArea}>
+        <Board players={players} snakes={SNAKES} ladders={LADDERS} />
+        <div className={styles.controls}>
+          <Dice value={diceValue} onRoll={rollDice} disabled={gameStatus.includes('wins')} />
+          <button className={styles.resetButton} onClick={resetGame}>Reset Game</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)